### DISCUSSION TITLE
MODIS LST output pixel scale/resolution do not match the native resolution

### DISCUSSION THREAD

**Discussion by arindan** [google/Xee#193] [Q&A] [answered]
Hi everyone,

I am facing output pixel resolution related issue while accessing MODIS LST data. MODIS has 1 km (1000 m) per pixel resolution (this product MOD11A2), but when I access the data without specifying `scale` within `xr.open_dataset()`, it gives me data that has very very large pixel size (see 1st figure below). Consideing, that the `scale` in `xr.open_dataset()` is in degrees, I make it `scale=0.01` (to get a roughly 1 km per pixel grid), it gives better and realistic-looking values per pixel (see 2nd figure below).

Here is a reproducible code, if anyone would like to try:
```
import ee
ee.Authenticate()
ee.Initialize(opt_url='https://earthengine-highvolume.googleapis.com')

ic = (
    ee.ImageCollection('MODIS/061/MOD11A2')
    .filterDate('2000-10-15', '2000-11-30')
    .select('LST_Day_1km')
)

region = ee.Geometry.Rectangle(72, 34, 80, 37.5)
ds = xr.open_dataset(ic,
                     engine='ee',
                     crs='EPSG:4326',
                     scale=0.01, # 1st try WITHOUT scale, 2nd try scale=0.01
                     geometry=region)

# plot
fig, ax = plt.subplots(figsize=(8,8))
ds.LST_Day_1km[0].T.plot(ax=ax, cbar_kwargs={'shrink': 0.3})
plt.show()
```

**My TWO questions are:**
**1. What would be the best method to get an output of ~1 km per pixel using xee-supported `xr.open_dataset()`?**
**2. If I use this method, specifying `scale=0.01`, is it is valid/good way to handle the pixel resolution?**

**Any solution would be super helpful.**

**Figure 1 (without specifying `scale`):**
![Image](https://github.com/user-attachments/assets/d5477653-da23-4332-8d36-9dcda04fc31f)

**Figure 2 (with `scale=0.01`):**
![Image](https://github.com/user-attachments/assets/d6017d39-d2a5-4faf-8576-7ceee319afa5)

**Comment by tylere** [comment, marked as answer]
Using the scale parameter can be a valid approach, but note that the pixel scale (m/degree) varies with latitude. See:
https://en.wikipedia.org/wiki/Latitude#Meridian_distance_on_the_ellipsoid
At the latitude of the example region (~36 deg N), using scale=0.01 degrees will result in a pixel size that is approximately 1.1 km (N-S) by 0.9 km (E-W).

If you want more control over the pixel sizing and grid origin, you could pass a specific projection to `xr.open_dataset()` using the `projection` parameter.

**Reply by arindan** [reply]
Thank you for the response. I appreciate it.
